lib/mongoose_push_web/plugs/cast_and_validate.ex

```diff
@@ -0,0 +1,133 @@
+defmodule MongoosePushWeb.Plug.CastAndValidate do
+  @behaviour Plug
```

Comment:
Does it make sense to add `moduledoc` to this plug?
https://hexdocs.pm/elixir/writing-documentation.html

Reply:
It's a wrapper for already documented `OpenApiSpex.Plug.CastAndValidate` module, so I would find more than 1 line doc reduntant here.

Reply:
So indlude the one needed doc line please.